app/services/identity_linker.rb

```diff
@@ -92,4 +92,9 @@ def merge_attributes(verified_attributes)
     verified_attributes = verified_attributes.to_a.map(&:to_s)
     (identity.verified_attributes.to_a + verified_attributes).uniq.sort
   end
+
+  def service_provider
+    return if provider.blank?
+    @service_provider ||= ServiceProvider.find_by(issuer: provider)
```

Comment:
I would consider renaming "provider" to "issuer" in the whole class, it's not a name we use anywhere else, did a double take because I was surprised at what it was

Reply:
Done in e22242e9e